src/components/accounts/recovery_setup/SetupRecoveryMethod.js

```diff
@@ -114,6 +122,7 @@ class SetupRecoveryMethod extends Component {
     }
 
     handleSendCode = async () => {
+        Mixpanel.track("SR Resend code", {resend_detail: this.method})
```

Comment:
This could be either `send` or `resend` as it's used in both scenarios. You can catch the `resend` event on line `281`

Reply:
will change to send code instead of resend code